package.json

```diff
@@ -75,7 +77,7 @@
     "@types/leaflet": "1.4.0",
     "@types/loadable__component": "5.10.0",
     "@types/loadable__server": "5.9.1",
-    "@types/node": "10.12.2",
+    "@types/node": "10.17.3",
```

Comment:
This resolved a weird `duplicate identifier IteratorResult` error that was fixed by upgrading `@types/node`.